src/pallene/c_compiler.lua

```diff
@@ -18,6 +18,7 @@ local c_compiler = {}
 
 local CC       = os.getenv("CC")       or "cc"
 local CFLAGS   = os.getenv("CFLAGS")   or "-O2"
+local SHLIBLOC = os.getenv("SHLIBLOC") or "/usr/local/lib"
```

Comment:
Pallene is installed via Luarocks, which checks that ptracer is installed. If Luarocks only checks default installation locations, then there would be nothing to gain by adding this configuration flag. And if we do need a configure flag, we should follow whatever scheme Luarocks uses (presumably it would use a ptracer-specific variables instead of a global sharedlib variable).

I think that for now we can just leave this out and assume people will install to default locations. We can add configuration later if it comes up.